Issue #403: Malware notifications (#465)
elastic/security-docs: docs/getting-started/install-endpoint.asciidoc

5. (Windows Only) On Window's 10 or above, you can use Elastic Security as your antivirus software. For more information, see <<register-as-antivirus, Register as antivirus>>
6. Click **Save** to save changes to the Policy.
7. On the dialog that appears, click **Save and Deploy changes**. If successful, a "Success" confirmation appears in the lower-right corner.

[role="screenshot"]
image::images/install-endpoint/malware-protection.png[]

[discrete]
[[register-as-antivirus]]
## Register as antivirus (Windows Only)

If you download the Elastic Agent on Windows 10 or above, you can configure Elastic Security as your antivirus software by doing the following:

1. Go to **Fleet** > **Agents**. A list of hosts with the Elastic Agent appears. 
2. Select your desired host from the list.
3. From the host's page, find your named **Endpoint** integration and select it. 
4. On the Integration Edit page, look for the **Settings** section and find **Type: Register as antivirus**. Toggle this option to enable. 
+
[role="screenshot"]

> image::images/register-as-antivirus.png[]

Reviewer: I'm a little lost here. Why do they need to do all this when they can select the option at the bottom? I can understand if they're in the Fleet app, but if they are following this page, they'll already be on the Policy config page. 

![image](https://user-images.githubusercontent.com/57149392/105898512-8944be80-5fe7-11eb-8f7c-17a3f38c83ff.png)

Author: These were written in a separate PR initially, but I was writing the content I discovered that this should be grouped together. Consolidating these in my next commit.

[role="screenshot"]
image::images/install-endpoint/malware-protection.png[]

[discrete]
[[malware-notifications]]
== Malware notications

> Platinum and enterprise users can customize the notifications on their host devices. These notifications appear whenever malware is detected or prevented with the syntax `Elastic Security {action} {filename}`.

Reviewer: ```suggestion
Notifications appear on protected hosts whenever malware is detected or prevented. Platinum and enterprise users can customize these notifications using the `Elastic Security {action} {filename}`syntax. 
```

> Policy actions determine default notification behaviors: 

Reviewer: ```suggestion
Policy configuration determines default notification behavior: 
```

> - **Prevent**: Notifications appear by default. Deselect the **Notify User** option to hide them.

Reviewer: ```suggestion
- **Prevent**: Notifications appear by default. Deselect the **Notify User** option to disable them.
```

> - **Detect**: Notifications do not appear. Select the **Notify User** option to begin receiving them.

Reviewer: ```suggestion
- **Detect**: Notifications do not appear. Select the **Notify User** option to enable them.
```

> If the malware policy is not selected, notifications will not appear on the host device.

Reviewer: ```suggestion
If malware protection is disabled, notifications will not appear on the host device.
```
